src/style.css

```diff
@@ -0,0 +1,105 @@
+html {
+  font-family: system-ui;
+}
+
+body {
+  background-color: rgb(243, 242, 242);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  overflow: hidden;
```

Comment:
I suggest that you shouldn't hide the overflow property of the body, and if you will, you should indicate if it is overflow-x or overflow-y otherwise, the overflow property of the body tag should be scroll 😃.